raycicmd/job_node.go

```diff
@@ -0,0 +1,19 @@
+package raycicmd
+
+// jobNode is a node for a job node or a group in the pipeline.
```

Comment:
is a job map to a buildkite build? `node for a job node` is a bit hard to understand

Reply:
`group` -> `group of steps`